api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,337 @@
+            - package: mango
+              version: 2.1
+
+    FreeUserBatch:
+      title: Vendor Public Recommendation
+      type: object
+      properties:
+        package:
+          type: string
+          example: "django"
+        version:
+          type: string
+          example: "1.1"
+        recommended_versions:
+          type: string
+          example: "3.1"
+        vulnerability:
+          type: array
+          items:
+            $ref: '#/components/schemas/VulnerabilityDetailsBatch'
+
+    VulnerabilityDetailsBatch:
+      title: Vulnerability Details Batch
+      description: Information of the vulnerability found
+      type: object
+      properties:
+        id:
+          type: string
+          example: "SNYK-1234"
+        severity:
+          type: string
+          example: "medium"
+        title:
+          type: string
+          example: "Information Exposure"
+        url:
+          type: string
+          example: "https://snyk.io/vuln/SNYK-PYTHON-DJANGO-40055"
+        fixed_in:
+          type: array
+          items:
+            type: string
+          example:
+            - "1.1.3"
+            - "1.2.4"
+
+    VulnerabilityDetails:
+      title: Vulnerability Details
+      description: Information of the vulnerability found
+      type: object
+      properties:
+        vulnerability:
+          type: array
+          items:
+            type: object
+            properties:
+              vendor_cve_ids:
+                type: string
+                example: "SNYK-1234"
+              severity:
+                type: string
+                enum:
+                  - low
+                  - medium
+                  - high
+                  - critical
+              title:
+                type: string
+                example: SQL Injection
+              url:
+                type: string
+                example: https://snyk.io/vuln/SNYK-PYTHON-DJANGO-4002
+              fixed_in:
+                type: array
+                items:
+                  type: string
+                example:
+                  - ">=0.19.0-beta.1"
+
+    FreeUser:
+      title: Vendor Public Recommendation
+      type: object
+      properties:
+        package:
+          type: string
+          example: "django"
+        version:
+          type: string
+          example: "1.1"
+        recommendation:
+          type: object
+          properties:
+            recommended_version:
+              type: string
+              example: "3.1"
+            component_analyses:
+              $ref: '#/components/schemas/VulnerabilityDetails'
+
+    NoVulnerabilities:
+      title: No Vulnerability Response
+      type: object
+      properties:
+        package:
+          type: string
+          example: "django"
+        version:
+          type: string
+          example: "1.1"
+        recommendation:
+          type: string
+          example: "No vulnerabilities found for package"
```

Comment:
"No vulnerabilities found for package" -> "No vulnerabilities found for the package"